sims/verilator/Makefile

```diff
@@ -1,112 +1,207 @@
-#########################################################################################
+#############################################################################
 # verilator makefile
-#########################################################################################
+#############################################################################
 ifeq ($(shell which verilator),)
 $(error Did not find Verilator in PATH. Make sure all requirements are installed)
 endif
 
-#########################################################################################
+.PHONY: default debug compile compile-debug
+default: compile
+debug: compile-debug
+
+#############################################################################
 # general path variables
-#########################################################################################
+#############################################################################
 base_dir=$(abspath ../..)
 sim_dir=$(abspath .)
 
-#########################################################################################
+#############################################################################
 # include shared variables
-#########################################################################################
+#############################################################################
 include $(base_dir)/variables.mk
 
-#########################################################################################
+#############################################################################
 # name of simulator (used to generate *.f arguments file)
-#########################################################################################
+#############################################################################
 sim_name = verilator
 
-#########################################################################################
-# vcs simulator types and rules
-#########################################################################################
+#############################################################################
+# verilator simulator types and rules
+#############################################################################
 sim_prefix = simulator
 sim = $(sim_dir)/$(sim_prefix)-$(MODEL_PACKAGE)-$(CONFIG)
 sim_debug = $(sim_dir)/$(sim_prefix)-$(MODEL_PACKAGE)-$(CONFIG)-debug
 
+# verilator doesn't use +permissive, but common.mk expects the simulator's
```

Comment:
This is incorrect

Reply:
how

Reply:
colin's PR was going to fix this.

Reply:
This statement is not true. `+permissive` is a valid HTIF argument, and is currently able to be passed to verilator if you set it so, so this comment is wrong, and there's no reason to add it.